capa/features/extractors/ida/file.py

```diff
@@ -134,7 +134,7 @@ def extract_file_section_names() -> Iterator[Tuple[Feature, Address]]:
         yield Section(idaapi.get_segm_name(seg)), AbsoluteVirtualAddress(seg.start_ea)
 
 
-def extract_file_strings() -> Iterator[Tuple[Feature, Address]]:
+def extract_file_strings(ctx=None) -> Iterator[Tuple[Feature, Address]]:
```

Comment:
```suggestion
def extract_file_strings(ctx) -> Iterator[Tuple[Feature, Address]]:
```

Reply:
mypy gives an error if I don't add a default value for ctx here but I am not sure why.
I think it is related to this issue: https://github.com/python/mypy/issues/9527
`capa/features/extractors/ida/file.py:189: error: Cannot call function of unknown type  [operator]`
`Found 1 error in 1 file (checked 53 source files)`